atomic_reactor/utils/remote_host.py

```diff
@@ -360,6 +360,22 @@ def is_free(self, slot_id: int) -> bool:
             slot = HostSlot(self, session, slot_id)
             return slot.is_free
 
+    def prid_in_slot(self, slot_id: int) -> Optional[str]:
+        """ Check which prid is in the slot
+
+        :param slot_id: int, slot ID
+        :return: prid string
+        :rtype: str
+        """
+        if not self._is_valid_slot_id(slot_id):
+            return None
+
+        # We don't need to lock the slot to check whether it's free,
+        # so using a normal ssh session is good enough
+        with self._ssh_session() as session:
+            slot = HostSlot(self, session, slot_id)
```

Comment:
I believe HostSlot class expects host as well https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/utils/remote_host.py#L448

Reply:
host is self, the same as in 'is_free' above

btw this is working code